### ISSUE TITLE
[🐛] Banner Ads not loading on React Native 0.72.3 (Android)

### ISSUE THREAD

**Issue by lck-ams** [invertase/react-native-google-mobile-ads#421]
### What happened?

Banner Ads are not loading in my App, even with Test Ads. Other type of Ads works fine (e.g. Interstitial), but no request were sent for Banner Ads when looking at Ad Inspector.

Ads were working when it was in React Native 0.72.1.

### Platforms

Android, but have not tested behavior on iOS

### React Native Info

```Shell
N/A
```

### Are your using Typescript?

- [ ] My project is using Typescript

### package.json

```JSON
"expo": "49.0.6",
"react": "18.2.0",
"react-native": "0.72.3",
"react-native-google-mobile-ads": "11.6.1"
```

### app.json

```JSON
"react-native-google-mobile-ads": {
  "android_app_id": "ca-app-pub-xxxxxxxxxxxx~xxxxxxxxxxx",
  "ios_app_id": "ca-app-pub-xxxxxxxxxxxx~xxxxxxxxxxx",
}

Both App Id are valid, the issue also affects Test Ads, so it's not Production specific.
```

### ios/Podfile

```Ruby
N/A
```

### android/build.gradle

```Groovy
N/A
```

### android/app/build.gradle

```Groovy
N/A
```

### android/settings.gradle

```Groovy
N/A
```

### AndroidManifest.xml

```XML
N/A
```

**Comment by dylancom** [comment]
What does the callback `onAdFailedToLoad` log?

**Comment by lck-ams** [comment]
It logs nothing on `onAdFailedToLoad`, `onAdLoaded`, or `onAdOpened`. It does however takes up the banner space, but it just displays a blank space.

**Comment by peter-featherstone** [comment]
Same. They aren't showing on iOS either.

**Comment by saravanakumargn** [comment]
```
    "react-native-google-mobile-ads": "^11.6.4",
    "react-native": "0.72.3",
    "expo": "~49.0.6",
```
It's showing only blank space. No error in error callbacks.

**Comment by saravanakumargn** [comment]
Please refer https://github.com/invertase/react-native-google-mobile-ads/issues/427#issuecomment-1666442152

**Comment by dylancom** [comment]
Should be fixed in 11.6.5. Thanks @saravanakumargn for giving more details.
